pyproject.toml

```diff
@@ -11,25 +11,25 @@ packages = [{ include = "pylenium" }]
 pylenium = "pylenium.scripts.cli:app"
 
 [tool.poetry.dependencies]
-python = "^3.7"
-pydantic = "1.9.0"
-requests = "2.27.1"
-pytest-xdist = "2.4.0"
+python = ">=3.8.1,<4.0"
```

Comment:
Isn't it easier to just use the semver specifier?
```
python = "^3.8.1"
```

Reply:
Good catch. I didn't test with `>=3.8.1` to see if any of the dependencies required the `<4.0`, which I'm assuming they wouldn't since `>=3.7.1` worked before. I'll get a patch going for this 👍🏽

Reply:
https://github.com/ElSnoMan/pyleniumio/pull/306